app/controllers/vendor_api/vendor_api_controller.rb

```diff
@@ -8,6 +8,17 @@ class VendorApiController < ActionController::API
     before_action :set_cors_headers
     before_action :require_valid_api_token!
 
+    def audit_user
+      return nil unless @metadata.present?
+
+      @audit_user ||= VendorApiUser.find_or_create_by(
+        email_address: @metadata.attribution.email,
+        full_name: @metadata.attribution.full_name,
+        user_id: @metadata.attribution.user_id,
```

Comment:
Should we namespace this a bit? `vendor_user_id`, maybe? just to disambiguate from the `user_id` in the audits table